quickstart/test/compose-test.zsh

```diff
@@ -153,7 +155,6 @@ docker compose exec ziti-controller \
     '
         # TODO: re-add cert checks to above test suite after https://github.com/openziti/ziti/pull/1278
         # zsh /persistent/check-cert-chains.zsh;
-docker compose run quickstart-test
+docker compose --profile test run --rm quickstart-test
```

Comment:
move test to a compose profile so it only runs when deliberately invoked, not every time the project is `up`'d